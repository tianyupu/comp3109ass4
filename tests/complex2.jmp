  a = 2;
  b = 1;
  d = a + b;
  goto L3;
L2:
  c = a + d;
  return c;
L3:
  a = a + 1;
  test = a > 100;
  a = 15;
  if test goto L2;
  d = 3;
  if d goto L5;
L5:
  b = b +1;
  test = b > 100;
  if test goto L2;
  goto L3;

