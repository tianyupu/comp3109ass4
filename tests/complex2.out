  a = 2;
  b = 1;
  d = a + b;
  a = a + 1;
  test = a > 100;
  a = 15;
  if test goto L2;
  d = 3;
  b = b +1;
  test = b > 100;
  if test goto L2;
  goto L3;
L2:
  c = a + d;
  return c;
